kolibri/plugins/management/assets/src/device_management/views/index.vue

```diff
@@ -49,8 +56,14 @@
     vuex: {
       getters: {
         pageName: ({ pageName }) => pageName,
+        welcomeModalVisible: ({ welcomeModalVisible }) => welcomeModalVisible,
         canManageContent,
       },
+      actions: {
+        hideWelcomeModal(store) {
```

Comment:
We don't really do this, but there didn't seem to be an existing file where this action would fit in, and it seemed silly to create an entire new file for one. Thoughts?

Reply:
This seems fine to me.